onnx_tf/handlers/backend/einsum.py

```diff
@@ -0,0 +1,14 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+
+@onnx_op("Einsum")
+
+class Einsum(BackendHandler):
+
+  @classmethod
+  def version_12(cls, node, **kwargs):
+    equation = node.attrs.pop("equation", "")
```

Comment:
Any reasons for 'pop'? If the purpose is to get the value, 'get' should do the job.

Reply:
@chinhuang007 the pop is required when I use cls.make_tensor_from_onnx_node to create this tf node. However cls.make_tensor_from_onnx_node cannot send in the the rest of the inputs in correct format, so I switch to call tf.einsum directly. Hence I don't need to pop the "equation" attribute anymore, I will change it back to get.